mobile-save-for-later/src/main/scala/com/gu/sfl/lambda/FetchArticlesLambda.scala

```diff
@@ -1,37 +1,36 @@
 package com.gu.sfl.lambda
 
 import com.amazonaws.services.cloudwatch.AmazonCloudWatchAsyncClientBuilder
-import com.gu.AwsIdentity
 import com.gu.sfl.Logging
 import com.gu.sfl.controller.FetchArticlesController
 import com.gu.sfl.identity.{IdentityConfig, IdentityServiceImpl}
+import com.gu.sfl.lambda.FetchArticlesConfig.{app, stage}
 import com.gu.sfl.lib.Parallelism.largeGlobalExecutionContext
-import com.gu.sfl.lib.{CloudWatchImpl, GlobalHttpClient, SsmConfig}
+import com.gu.sfl.lib.{CloudWatchImpl, GlobalHttpClient}
 import com.gu.sfl.persistence.{PersistenceConfig, SavedArticlesPersistenceImpl}
 import com.gu.sfl.savedarticles.FetchSavedArticlesImpl
 
+object FetchArticlesConfig {
+  private val identityApihostKey = "IdentityApiHost"
+  private val stageKey = "Stage"
+  private val appKey = "App"
+
+  lazy val identityApiHost = sys.env.getOrElse(identityApihostKey, throw new NullPointerException(identityApihostKey))
```

Comment:
I think the exception needs to be a bit more clear. Something like s"Couldn't load environment variable $key"